test/test_torchrun.py

```diff
@@ -0,0 +1,91 @@
+# Owner(s): ["module: unknown"]
+
+from torch.testing._internal.common_utils import TestCase, run_tests
+import os
+import shutil
+import subprocess
+
+class TestTorchrun(TestCase):
+    def test_jemalloc(self):
```

Comment:
thanks for adding the test! could we move this to `test/utils/inference/run_test.py` to follow the test directory conventions as the rest of the project?

Reply:
Sure.